src/vaadin-pro-grid-inline-editing-mixin.html

```diff
@@ -77,6 +80,14 @@
       this.notifyPath('items.' + index + '.' + path, value);
     }
 
+    _isAlphaNumericKey(keyCode) {
```

Comment:
`keyCode` is deprecated, consider using `key` instead.

Reply:
what do you thing regarding iOS 9 not supporting `key`? Is it safe there, as the user won't be able to focus the cell from mobile keyboard to type in it anyways?

Reply:
It doesn't make much sense to use deprecated platform features just because of iOS9 which AFAIK we will stop supporting soon.

Editable grid on iOS 9 is in reality 0.00% of use-cases.

Reply:
Actually, we use `key` in list-box: https://github.com/vaadin/vaadin-list-mixin/blob/e489859b5d904c49ccb17a88d7f3184b5b9019ac/vaadin-list-mixin.html#L127

And I don't expect `keydown` to fire on IOS 9. So, I guess it's ok to use it here, too.